CMakeLists.txt

```diff
@@ -835,7 +834,8 @@ qore_user_module("qlib/WebSocketHandler.qm" "HttpServerUtil;WebSocketUtil")
 qore_user_module("qlib/WebUtil.qm" "HttpServerUtil;Mime;Util")
```

Comment:
BulkSqlUtil, FreetdsSqlUtil, MysqlSqlUtil, OracleSqlUtil, PgsqlSqlUtil, TableMapper, TelnetClient should go to base 3.
HttpServer, Pop3Client, SchemaReverse, SewioWebSocketClient, SmtpClient, WebSocketHandler, WebUtil should go to base 4.

Reply:
Bases are tricky as dependency in CMakefile is not reliable as we also need avoid doxygen warnings (@ref does not exist). So I would not reshuffle it now and continue getting rid with warnings

Reply:
The cmake macro qore_user_module creates dependencies between the doc modules (just like they are written in the second parameter). Can you show an example where they are not working?